invenio_accounts/utils.py

```diff
@@ -216,3 +239,33 @@ def validate_username(username):
         # text explaining the validation rules.
         message = current_app.config["ACCOUNTS_USERNAME_RULES_TEXT"]
         raise ValueError(message)
+
+
+def validate_domain_form(form, field):
```

Comment:
Will be needed in user-profiles to check the domain when a user want to change their email address.